stories/rating-indicator/rating-indicator.stories.js

```diff
@@ -0,0 +1,433 @@
+                <label class="fd-rating-indicator__label" for="rating-compact-1"></label>
+                
+                <input aria-label="2 star" type="radio" class="fd-rating-indicator__input" id="rating-compact-2" name="rating-compact" value="2" checked>
+                <label class="fd-rating-indicator__label" for="rating-compact-2"></label>
+                
+                <input aria-label="3 star" type="radio" class="fd-rating-indicator__input" id="rating-compact-3" name="rating-compact" value="3">
+                <label class="fd-rating-indicator__label" for="rating-compact-3"></label>
+                
+                <input aria-label="4 star" type="radio" class="fd-rating-indicator__input" id="rating-compact-4" name="rating-compact" value=4">
+                <label class="fd-rating-indicator__label" for="rating-compact-4"></label>
+                
+                <input aria-label="5 star" type="radio" class="fd-rating-indicator__input" id="rating-compact-5" name="rating-compact" value="5">
+                <label class="fd-rating-indicator__label" for="rating-compact-5"></label>
+            </div>
+            <span class="dynamic-text">(2 of 5)</span>
+        </div>
+    </div>
+    
+    <div class="example-container">
+        <span>Condensed:</span>
+        <div class="fd-rating-indicator fd-rating-indicator--condensed">
+            <div class="fd-rating-indicator__container" aria-label="Rating of this item is 2 out of 5">
+                <input aria-label="1 star" type="radio" class="fd-rating-indicator__input" id="rating-condensed-1" name="rating-condensed" value="1">  
+                <label class="fd-rating-indicator__label" for="rating-condensed-1"></label>
+                
+                <input aria-label="2 star" type="radio" class="fd-rating-indicator__input" id="rating-condensed-2" name="rating-condensed" value="2" checked>
+                <label class="fd-rating-indicator__label" for="rating-condensed-2"></label>
+                
+                <input aria-label="3 star" type="radio" class="fd-rating-indicator__input" id="rating-condensed-3" name="rating-condensed" value="3">
+                <label class="fd-rating-indicator__label" for="rating-condensed-3"></label>
+                
+                <input aria-label="4 star" type="radio" class="fd-rating-indicator__input" id="rating-condensed-4" name="rating-condensed" value=4">
+                <label class="fd-rating-indicator__label" for="rating-condensed-4"></label>
+                
+                <input aria-label="5 star" type="radio" class="fd-rating-indicator__input" id="rating-condensed-5" name="rating-condensed" value="5">
+                <label class="fd-rating-indicator__label" for="rating-condensed-5"></label>
+            </div>
+            <span class="dynamic-text">(2 of 5)</span>
+        </div>
+    </div>
+`;
+
+customIconSizes.parameters = {
```

Comment:
Since this is for all the sizes, I would rename this section "Sizes"

Reply:
fixed